File: week02/day03/src/Greet.java
/**
 * Created by Bálint on 2017. 03. 22..
 */
public class Greet {
  //  From now on, create the usual class wrapper
  //  and main method on your own.

  // - Create a string variable named `al` and assign the value `Greenfox` to it
  // - Create a function called `greet` that greets it's input parameter
  //     - Greeting is printing e.g. `Greetings dear, Greenfox`
  // - Greet `al`
  public static void main(String[] args) {
    String al = "Greenfox";
    System.out.println(greetFunction(al));
  }

  public static String greetFunction(String greet) {
    return "Greetings dear, " + greet;

  }
}

File: javascript_practice/2-functions-and-arrays/greet.js
'use strict';
// - Create variable named `al` and assign the value `Greenfox` to it
// - Create a function called `greet` that greets it's input parameter
//     - Greeting is printing e.g. `Greetings, dear Greenfox`
//     - Prepare for the special case when no parameters are given
// - Greet `al`

var al = 'Greenfox';

function greet(name='friend') {
    console.log('Greetings, dear ' + name);
}

function greet2(name='friend'){
    return 'Greetings, dear ' + name;
}

greet(al);
console.log(greet2(al));
